source/intro.tex

```diff
@@ -424,8 +424,8 @@
 
 \pnum
 Throughout this International Standard, each example is introduced by
-``\begin{example}'' and terminated by ``\end{example}''. Each note is
-introduced by ``\begin{note}'' and terminated by ``\end{note}''. Examples
+``[\,\textit{Example:}'' and terminated by ``\textit{\,---\,end example}\,]''. Each note is
+introduced by ``[\,\textit{Note:}'' and terminated by ``\textit{\,---\,end note}\,]''. Examples
```

Comment:
That's an excellent idea. Alternatively, you can factor the lexical components out into a separate macro.